build.py

```diff
@@ -256,6 +269,17 @@ def _linux_install(package: Package) -> Generator[None, None, None]:
         yield
 
 
+def _linux_get_archs(file: str) -> set[str]:
+    # TODO: this could be more accurate
+    out = subprocess.check_output(("file", file)).decode()
+    if ", x86-64," in out:
```

Comment:
I'm sure nothing will break one day :)

Reply:
oh 100% this will have to be a better method at some point (`elftools` maybe?) but for now it's good enough -- also from the things I've seen the mistake only gets made on macos and not on linux